addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/WebSocketChannelDTO.java

```diff
@@ -31,30 +31,30 @@
 public class WebSocketChannelDTO implements Comparable<WebSocketChannelDTO> {
 
     /** ChannelId. */
-    public Integer id;
+    private Integer id;
 
     /** Is not necessarily the same as the {@link WebSocketChannelDTO#url}. */
-    public String host;
+    private String host;
 
     /** Port where this channel is connected at. Usually 80 or 443. */
-    public Integer port;
+    private Integer port;
 
     /** URL used in HTTP handshake. */
-    public String url;
+    private String url;
 
     /** Timestamp taken, when connection was established successfully. */
-    public Long startTimestamp;
+    private Long startTimestamp;
 
     /** Timestamp of close, otherwise <code>null</code>. */
-    public Long endTimestamp;
+    private Long endTimestamp;
 
     /** Id of handshake message. */
-    public Integer historyId;
+    private Integer historyId;
 
     public WebSocketChannelDTO() {}
 
     public WebSocketChannelDTO(String host) {
-        this.host = host;
+        this.setHost(host);
```

Comment:
This seems unnecessary (same for other uses in the class itself).

Reply:
SonatLint auto changed that, will revert.